code/workspaces/web-app/src/auth/auth.js

```diff
@@ -79,16 +79,14 @@ function processHash(authResponse) {
 }
 
 function processResponse(authResponse) {
-  const state = undefined;
-  const appRedirect = state ? state.appRedirect : undefined;
-  const expiresAt = authResponse.expiresAt || expiresAtCalculator(authResponse.expires_at);
+  const appRedirect = undefined;
+  const expiresAt = authResponse.expires_at || expiresAtCalculator(authResponse.expires_at);
   const identity = authResponse.identity || processIdentity(authResponse.profile);
 
   return {
     ...authResponse,
     appRedirect,
```

Comment:
Do we need to add appRedirect if it is always going to be undefined. Isn't that the same as it not being there? Could just get rid of it like `state`?

Reply:
That makes sense, have just been looking back at where we need to specify appredirect for direct-linking to work but I think it's handled by the library now.